### ISSUE TITLE
Extension won't work on non-english languages

### ISSUE THREAD

**Issue by vrcca** [stephenlewis/no-read-receipts#1]
Extension won't work on non-english languages.

I'd change this part:
 if(text.match(/read receipts requested/i)) {
          node.click();
        }

To:
node.click();

**Comment by stephenlewis** [comment]
Good point (and sorry I've taken so long to reply)! The reason that check is in there is that I felt clicking any button labelled 'Later' in the Google Mail UI would be dangerous - I wanted to make sure that it was sitting inside the correct confirmation modal.

How would you feel about having a configuration UI that asked you for the text to match (for both the 'read receipts requested' and 'later')? I'd happily incorporate the appropriate translations into the extension, but I can't think of a way to find out what they are in one batch - Google serve up different JS for different locales, with the appropriate strings already included. If you've got any ideas for how to do that, I'll give it a go.

**Comment by SamSamFR** [comment]
Hi

This extension is simple, but a great help to reduce daily hassles.
I too had to change my gmail language setting to english for it to work.

If you're still maintaining this, could you look up the text to match from a database of strings (based on locale, or just match any one of them).

You can start with this one for french locales :
/Confirmations de lecture demandées/
![image](https://user-images.githubusercontent.com/5165588/75446287-676e3200-5967-11ea-8395-4d7d4424dd96.png)

**Comment by vrcca** [comment]
@stephenlewis Oops! It's been so long! I no longer use either Chrome or Gmail hehe

As for the proposal, I guess a configurable config would be great. Or maybe it could try to match with the button/form id?
